src/components/common/BackButton.tsx

```diff
@@ -0,0 +1,21 @@
+import { ReactComponent as IconChevronLeft } from 'assets/icons/chevron-left.svg';
+import { cn } from 'utils/helpers';
+
+interface Props {
+  onClick?: () => void;
+  className?: string;
+}
+
+export const BackButton = ({ onClick, className }: Props) => {
+  return (
+    <button
```

Comment:
add the `type="button"` to prevent it to submit if in a `form`

Reply:
done